man/org.freedesktop.oom1.xml

```diff
@@ -56,12 +59,16 @@ node /org/freedesktop/oom1 {
 
     <variablelist class="dbus-method" generated="True" extra-ref="DumpByFileDescriptor()"/>
 
+    <variablelist class="dbus-signal" generated="True" extra-ref="Killed"/>
+
     <!--End of Autogenerated section-->
 
     <refsect2>
       <title>Methods</title>
 
       <para>...</para>
+
+      <para><function>Killed</function> signal is sent when any cgroup is killed by oomd</para>
```

Comment:
Please be more precise here. Describe the two possible values of the 'reason' parameter and what they mean.

Reply:
```Possible reasons:
      <literal>memory-used</literal> - application took too much memory</para>
      <literal>memory-pressure</literal> - application took enough memory to cause sufficient slowdown of other applications</para>```

I suggest this (AFAIK what the PSI is)